kiwi/storage/loop_device.py

```diff
@@ -98,12 +99,13 @@ def create(self, overwrite: bool = True):
         )
         self.node_name = loop_call.output.rstrip(os.linesep)
 
-    def __del__(self):
+    def __exit__(self, exc_type, exc_value, traceback):
         if self.node_name:
-            log.info('Cleaning up %s instance', type(self).__name__)
             try:
                 Command.run(['losetup', '-d', self.node_name])
-            except Exception:
-                log.warning(
-                    'loop device %s still busy', self.node_name
+            except Exception as issue:
+                log.error(
+                    'loop cleanup on {0} failed with: {1}'.format(
+                        self.node_name, issue
+                    )
```

Comment:
you don't like f-strings, do you :stuck_out_tongue: ?
```suggestion
                    f"loop cleanup on {self.node_name} failed with: {issue}'
                    )
```

Reply:
I like them, but I don't like the long lines and if there are many substitutions in it. In this case I think it's an easier read to use format() :smile_cat: